tests/migration_test.go

```diff
@@ -1013,28 +1047,235 @@ var _ = Describe("[rfe_id:393][crit:high[vendor:cnv-qe@redhat.com][level:system]
 				}, 180*time.Second, 500*time.Millisecond).Should(Equal(v1.MigrationSucceeded))
 			})
 
-			Context("with node tainted", func() {
+			Context("with node tainted during node drain", func() {
+				It("[test_id:2221] should migrate a VMI under load to another node", func() {
```

Comment:
Is it worth to add to this test some verification that VM was not rebooted or crashed during migration? Something simple, e.g., console does not ask for a password because we already logged in..

Reply:
yep, this is worth checking.

In this case we're verifying a VMI and not a VM. So, if the qemu process crashed, the VMI wouldn't still be active.  I'll add a check here to make sure the VMI has both moved to another node and has Status.Phase==Running.

Reply:
fixed